src/types.ts

```diff
@@ -27,6 +27,11 @@ export interface FunctionPlotOptionsAxis {
    * True to invert the direction of the axis
    */
   invert?: boolean
+
+  /**
+   * The position of the axis
```

Comment:
Please add some comments about the default values for both the xAxis and the yAxis